earth_enterprise/src/fusion/autoingest/plugins.src/RasterProject.src

```diff
@@ -181,7 +181,8 @@ class RasterProjectAssetVersionImplD {
   uint32 BuildIndex(
     const khTilespace &tilespace,
     const std::vector<InsetInfo<RasterProductAssetVersion> >& inset_infos,
-    const std::string& date);
+    const std::string& date/*,
+    InsetTilespaceIndex<InsetInfo<RasterProductAssetVersion>> &tilespaceIndex*/);
```

Comment:
Was this change meant to be checked in? It's just commented out.

Reply:
Oops. That was left over from the initial stab I took at it. I'll take that out.